### PR TITLE
Bugfix/usb timeout

### PR DESCRIPTION
**Pull request by christomaszewski**

### CODE REVIEW

**Comment by christomaszewski** [comment]
We good to merge this?

**Comment by psigen** [comment]
Apparently that motor comment is already in `master`, we should probably leave it in, just in case it breaks something.

**Comment by christomaszewski** [comment]
I am doing the multiple motor enable methods you suggested last night. I just want to merge to master and then i'll make a separate branch for the power fix

File: firmware/firmware.ino [hunk 1]
```diff
@@ -467,6 +481,7 @@ void motorUpdateLoop()
       platypus::Motor* motor = platypus::motors[motor_idx];
       motor->velocity(motor->velocity() * 0.8);
     }
+    break;
```
Comment (psigen) [inline firmware/firmware.ino]: @christomaszewski: did you change this on purpose?

**Comment by psigen** [comment]
Cool, I just removed that from this commit.  Can you take a look at the comment about the `break`? I think everything else is good to go.

**Pull request merged**